ccbc-api/src/main/java/io/coin/ccbc/api/application/AddressApplicationService.java

```diff
@@ -71,29 +107,62 @@ private static double getMockKusdtPrice() {
 
   @Transactional(readOnly = true)
   public List<AddressAssetDto> getAssets(String address) {
-    return Arrays.asList(
-        new AddressAssetDto(
-            getMockKlayCoinDto(),
-            getMockKlayAmount(),
-            getMockKlayAmount(),
-            getMockKlayAmount(),
-            getMockKlayPrice()
-        ),
-        new AddressAssetDto(
-            getMockKethCoinDto(),
-            getMockKethAmount(),
-            getMockKethAmount(),
-            getMockKethAmount(),
-            getMockKethPrice()
-        ),
-        new AddressAssetDto(
-            getMockKusdtCoinDto(),
-            getMockKusdtAmount(),
-            getMockKusdtAmount(),
-            getMockKusdtAmount(),
-            getMockKusdtPrice()
-        )
+    // TODO : 매번 불러오지말고 얘도 캐시개념 도입 가능
+    List<Coin> allCoins = coinRepository.findAll();
+    List<Pool> allPools = poolRepository.findAllWithCoin();
+
+    Address walletAddress = Address.of(address);
+
+    // 풀 정보 조회를위한 맵
+    Map<Address, Pool> poolMap =
+        allPools.stream()
+            .collect(Collectors.toMap(Pool::getAddress, pool -> pool));
+
+    // 온체인 pool balance 데이터
+    List<WalletPoolBalance> poolBalances = blockchainClient.getPoolBalances(
+        allPools.stream().map(Pool::getAddress).toList(),
+        walletAddress
     );
+
+    // 온체인 wallet balance 데이터
+    List<WalletCoinBalance> coinBalances = blockchainClient.getCoinBalances(
+        allCoins.stream().map(Coin::getAddress).toList(),
+        walletAddress
+    );
+
+    // coin address 별로 result asset dto
+    Map<Address, AddressAssetDto> res = allCoins.stream()
+        .collect(
+            Collectors.toMap(Coin::getAddress, coin -> new AddressAssetDto(CoinDto.from(coin))));
+
+    // 이자 산정을 위한 ksp dto. todo :: airdrop 고민
+    AddressAssetDto kspAssetDto = res.get(Address.KSP_ADDRESS);
+
+    // pool balance 반영
+    poolBalances.forEach(walletPoolBalance -> {
+      Pool pool = poolMap.get(walletPoolBalance.getPoolAddress());
+      res.get(pool.getCoin0().getAddress()).addPoolAmount(walletPoolBalance.getToken0Balance());
+      res.get(pool.getCoin1().getAddress()).addPoolAmount(walletPoolBalance.getToken1Balance());
+      kspAssetDto.addInterestAmount(walletPoolBalance.getInterestAmount());
+    });
+
+    // wallet balance 반영
+    coinBalances.forEach(
+        walletCoinBalance -> res.get(walletCoinBalance.getCoinAddress())
+            .addWalletAmount(walletCoinBalance.getAmount())
+    );
+
+    // 데이터 없는것 제외 후 coin price 반영
+    return res.values()
+        .stream()
+        .filter(addressAssetDto ->
+            addressAssetDto.getDepositableAmount().signum() != 0 ||
+                addressAssetDto.getDepositedAmount().signum() != 0 ||
+                addressAssetDto.getInterestAmount().signum() != 0
```

Comment:
```suggestion
            addressAssetDto.getDepositableAmount().signum() != 0 ||
                addressAssetDto.getDepositedAmount().signum() != 0 ||
                addressAssetDto.getInterestAmount().signum() != 0
```

이것들 addressAssetDto 안에 함수 만들어서 ㄱㄱ

Reply:
앗 사실그랬는데 isEmpty했더니 필드로나와서 헤헤 disable해서 dto안에넣겟슴다